Functions.Templates/Templates/HttpTrigger-CSharp-Isolated-NetCore/.template.config/template.json

```diff
@@ -73,7 +73,6 @@
             "args": {
                 "referenceType": "package",
                 "reference": "Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore",
-                "version": "1.3.2",
```

Comment:
This removal is intentional and is required to prevent a package downgrade. If a reference is already in place, it will be preserved. If it is not, the system will grab latest stable.